CONTRIBUTING.md

```diff
@@ -178,6 +178,8 @@ We aim for [ASIL-D](https://en.wikipedia.org/wiki/Automotive_Safety_Integrity_Le
 [ISO26262](https://en.wikipedia.org/wiki/ISO_26262) is also a good read-up if you want to learn more about automotive
 safety. A nice introduction [video](https://www.youtube.com/watch?v=F4GzsA00s5I) was presented on CppCon 2019.
 
+If you want to report a vulnerability, please use the [Eclipse process](https://www.eclipse.org/security/).
```

Comment:
Something weird is going on here. These changes are already on the master branch.